packages/react-focus/src/components/FocusZone/FocusZone.tsx

```diff
@@ -155,11 +151,13 @@ export class FocusZone extends React.Component<IFocusZoneProps> implements IFocu
         _outerZones.add(this);
 
         if (windowElement && _outerZones.size === 1) {
-          _disposeGlobalKeyDownListener = on(windowElement, 'keydown', this._onKeyDownCapture, true);
+          windowElement.addEventListener('keydown', this._onKeyDownCapture, true);
```

Comment:
Can we use the library to add the global one-time listener, too? We're specifically avoiding the native API in order to handle small variations between browsers, still, as we still support IE 11.

Reply:
Interesting idea! Any suggestions for what the API would look like?